force-app/main/default/pages/lmsPublisherVisualforce.page

```diff
@@ -0,0 +1,40 @@
+<apex:page controller="LMSVisualforceController" docType="html-5.0">
+    <!-- <apex:includeScript value="{!URLFOR($Resource.lmsvf, 'lmsvf.js')}"/> -->
+    <apex:slds />
+    <div class="slds-card">
+        <div class="slds-card__body slds-card__body_inner">
+            <!-- Placeholder to build list. -->
+            <ul data-list="list"></ul>
+        </div>
+
+        <footer class="slds-card__footer">
+            <c:viewSource
+                source="pages/{!$CurrentPage.Name}.page"
+                branch="beta"
```

Comment:
A likely source of error and another reason to stick to LWC format for `viewSource`.